ux/damage_panel.go

```diff
@@ -43,17 +44,56 @@ func NewDamagePanel(entity *gurps.Entity) *DamagePanel {
 	})))
 	p.DrawCallback = func(gc *unison.Canvas, rect unison.Rect) { drawBandedBackground(p, gc, rect, 0, 2, nil) }
 
+	// Add basic thrust damage
 	p.AddChild(NewNonEditablePageFieldEnd(func(f *NonEditablePageField) {
 		f.SetTitle(p.entity.Thrust().String())
 		MarkForLayoutWithinDockable(f)
 	}))
 	p.AddChild(NewPageLabel(i18n.Text("Basic Thrust")))
 
+	// Add basic swing damage
 	p.AddChild(NewNonEditablePageFieldEnd(func(f *NonEditablePageField) {
 		f.SetTitle(p.entity.Swing().String())
 		MarkForLayoutWithinDockable(f)
 	}))
 	p.AddChild(NewPageLabel(i18n.Text("Basic Swing")))
 
+	// Add Lifting Strength-based damage
+	p.AddChild(NewNonEditablePageFieldEnd(func(f *NonEditablePageField) {
```

Comment:
This should probably be gated behind another setting as well, since standard rules don't typically use this and most likely most players won't need it.